README.rst

```diff
@@ -103,13 +113,13 @@ tools are on your path): ::
 
     mkdir build && cd build && cmake -DCMAKE_TOOLCHAIN_FILE=../etc/xmos_cmake_toolchain/xs3a.cmake -G"Ninja" ..
 
-Then to actually build the the library as a static binary just use the ``ninja`` command from the
+Then to actually build the library as a static binary just use the ``ninja`` command from the
 ``build`` directory.
 
 To include the unit tests and example applications in your build, use the following command
 instead: ::
 
-    mkdir build && cd build && cmake -DDEV_LIB_XCORE_MATH=1 -DCMAKE_TOOLCHAIN_FILE=../etc/xmos_cmake_toolchain/xs3a.cmake -G"Ninja" ..
+    mkdir build && cd build && cmake -DDEV_LIB_XCORE_MATH=1 -DCMAKE_TOOLCHAIN_FILE=../etc/xmos_cmake_toolchain/xs3a.cmake -G Ninja ..
```

Comment:
`DEV_LIB_XCORE_MATH` is already set as the project is top-level https://github.com/xmos/lib_xcore_math/blob/develop/CMakeLists.txt#L15
same for `-DCMAKE_TOOLCHAIN_FILE`

Reply:
I removed that option and rephrased the text, @uvvpavel, does it read ok for you?

Reply:
Yes, thank you